openid-4-verifiable-presentations-1_0.md

```diff
@@ -706,6 +724,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+This section provides guidance on processing and security considerations for the `attachments` parameter defined in (#new_parameters).
+
+### Proof of Possession
+
+In some cases, the Verifier may want to include attachments that are bound to a key or require verification of their origin.
+
+It is up to the ecosystem profile to define how proof of possession is conveyed. Verifiers SHOULD use the `nonce` parameter from the Authorization Request in any proof-of-possession calculation to bind to to this transaction. The Wallet MUST validate such proofs if defined by the profile and ignore or reject attachments that fail validation.
+
+### Multi-Relying Party Behavior
+
+In some cases, the Verifier may want to include attachments based on a specific eco system profile. In such a scenario, the Authorization Request MUST sent a signed request in the JWS JSON Serialization format, see (#jws-json-serialization).
```

Comment:
not entirely sure, but I think we should introduce the binding to client id (prefix) here
```suggestion
In some cases, the Verifier may want to include attachments based on specific ecosystem profiles bound to specific Client Identifier Prefixes. In such a scenario, the Authorization Request MUST sent a signed request in the JWS JSON Serialization format, see (#jws-json-serialization).
```

Reply:
good point to clarify it, I added it